teleop/htm/templates/userMenu/menu_logbox.html

```diff
@@ -1,26 +1,78 @@
-          <th>Speed Intervention</th>
-          <th>Latitude</th>
-          <th>Longitude</th>
-          <th>AP Steer</th>
-          <th>AP Brake</th>
-          <th>AP Steer Confidence</th>
-          <th>AP Brake Confidence</th>
-      </tr>
-  </thead>
-  <tbody>
-      <!-- Dynamic content will be loaded here -->
-  </tbody>
-</table>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Sora:wght@100..800&display=swap" rel="stylesheet">
+</head>
+
+<body class="logbox">
+    <div class="logbox" id="content">
+        <div>
+            <h1 class="logbox" id="title">Training Events</h1>
+        </div>
+
+        <div id="dropdown" class="logbox">
+            <label for="entries_nr">Show</label>
+            <select class="logbox" id="mySelect">
+                <option value="10">10</option>
+                <option value="25">25</option>
+                <option value="50">50</option>
+                <option value="100">100</option>
+            </select>
+            <span>entries</span>
+        </div>
+
+        <table id="display" class="logbox" style="width:100%">
+            <thead>
+                <tr>
+                    <th>ID</th>
+                    <th>Timestamp</th>
+                    <th>Status</th>
+                    <th>Mode</th>
+                    <th>Max Speed</th>
+                    <th>Desired Speed</th>
+                    <th>Velocity</th>
+                    <th>Steer</th>
+                    <th>Throttle</th>
+                    <th>Steer Intervention</th>
+                    <th>Speed Intervention</th>
+                    <th>Latitude</th>
+                    <th>Longitude</th>
+                    <th>AP Steer</th>
+                    <th>AP Brake</th>
+                    <th>AP Steer Confidence</th>
+                    <th>AP Brake Confidence</th>
+                </tr>
+            </thead>
+            <div id="table_order" class="logbox">
+            <tbody>
+                <!-- Dynamic content will be loaded here -->
+            </tbody>
+            </div>
+        </table>
+
+        <div id="undertable" class="logbox">
+            <div id="entries" class="logbox">
+                <p>Showing 
+                    <span class="logbox" id="from_nr">1</span>
+                     to 
+                    <span class="logbox" id="to_nr">10</span>
+                     of 
+                    <span class="logbox" id="max_nr">10</span>
+                     entries
+                </p>
+            </div>
+            <div id="pagination" class="logbox">
+                <button id="prevBtn" class="logbox limit">Previous</button>
+                <button class="logbox currentBtn" id="firstBtn">1</button>
+                <span id="bdots" class="logbox">...</span> 
```

Comment:
this isn't a good name for the var. What does `bdots` stands for? 
same applies for `edots`